deployments/examples/README.md

```diff
@@ -45,6 +45,9 @@ Deployment files for the driver and executor:
 * [driver](spark30/driver.yaml).
 * [executor](spark30/executor.yaml).
 
+## tfjob
+A simple example for [kubeflow/tensorflow](./tfjob/tf-job-mnist.yaml) image.
```

Comment:
Can we add a bit more doc about what does this job do?

Reply:
I have already add doc on it, do we need more?

Reply:
Sorry I think I've missed that in last review.